### ISSUE TITLE
[Feature] User Specified save location for edited scenarios

### ISSUE THREAD

**Issue by yegods** [NPBruce/valkyrie#1698]
## Is your feature request related to a problem? Please describe.**
I lost a whole scenario i was editing because I didn't know the scenario was only saved in some obscure location on windows... some hidden folder inside appdata, and definitely not some hidden folder that I don't know exists, and that doesn't get backed up when I backup my documents folder.  I would like to be able to specify where I save my scenarios.

## Describe the solution you'd like
Allow some configuration for where scenarios should be saved in the editor.  I don't really care how... file browser, configuration file entry, anything will do.

## Describe alternatives you've considered
i don't have any alternatives.  i won't be using the editor until I can ensure my edited scenarios are safe.

## Additional context
na

**Comment by vidimus** [comment]
@yegods , I see your point (it's a pity you lost all your work) and clearly it would be great to have this feature in the next Valkyrie version. 

Meanwhile, if you go to the editor, you can create a package from your scenario (tools-->create a package). That will create a folder in your desktop with the scenario files (the files are to export your scenario and publish it). The valkyrie extension file is basically a zip file containing your scenario files.

**Comment by yegods** [comment]
Thanks, I was wondering what that did, but since I don't look at my desktop very often, I never noticed the folder appear there.

**Comment by vidimus** [comment]
> Thanks, I was wondering what that did, but since I don't look at my desktop very often, I never noticed the folder appear there.

Yeah, I happened to me the same until I realize the app have created the folder there...

**Comment by Quantumrunner** [comment]
Duplicate of #1705

Released already a while ago:

See more information in wiki on [User settings](https://github.com/NPBruce/valkyrie/wiki/User-settings#userconfig)

>UserRoot | Override for the data storage location. Useful for portable installations or specific data paths. | Manual Only | D:\Games\ValkyrieData
